### PR TITLE
KAR-533

### PR DESCRIPTION
**Pull request by garima468**
KAR-533

### CODE REVIEW

File: project/ws/app/src/lib/routes/app-toc/components/app-toc-home/app-toc-home.component.ts [hunk 1]
```diff
@@ -1019,7 +1019,7 @@ export class AppTocHomeComponent implements OnInit, OnDestroy, AfterViewChecked,
   }
 
   downloadCert(certIdArr: any) {
-    if (certIdArr.length) {
+    if (certIdArr.length && certIdArr.length > 0) {
```
Comment (vishnubansaltarento) [inline project/ws/app/src/lib/routes/app-toc/components/app-toc-home/app-toc-home.component.ts]: Please put check for certIdArr as well

Comment (garima468) [reply project/ws/app/src/lib/routes/app-toc/components/app-toc-home/app-toc-home.component.ts]: pushed the changes

File: library/ws-widget/collection/src/lib/card-content-v2/card-content-v2.component.ts [hunk 2]
```diff
@@ -381,8 +381,9 @@ export class CardContentV2Component extends WidgetBaseComponent
       })
     if(certificateData.issuedCertificates.length > 0) {
       this.downloadCertificateLoading = true
-      let certData: any = certificateData.issuedCertificates[0]
-      this.certificateService.downloadCertificate_v2(certData.identifier).subscribe((res: any)=>{
+      let certData: any = certificateData.issuedCertificates
+      certData.sort((a: any, b: any) => new Date(b.lastIssuedOn).getTime() - new Date(a.lastIssuedOn).getTime())
+      this.certificateService.downloadCertificate_v2(certData[0].identifier).subscribe((res: any)=>{
```
Comment (vishnubansaltarento) [inline library/ws-widget/collection/src/lib/card-content-v2/card-content-v2.component.ts:386]: please raise this PR for 4.8.16

**Comment by vishnubansaltarento** [comment]
wrong PR

**Pull request closed without merging**